### DISCUSSION TITLE
Why github copilot is piad??

### DISCUSSION THREAD

**Discussion by Ghalbeyou** [community/community#19925] [Copilot Conversations]
Hi!

Why github copilot is paid???

**Comment by baysideengineer** [comment]
Hey @Ghalbeyou, good question that may can lead to a thousand answers. It is part of the business model of GitHub to make profit out of their products. It also is a direct competitor to [Amazon CodeWhisperer](https://aws.amazon.com/codewhisperer/) and [tabnine](https://www.tabnine.com). It seems there is a huge market for things like that and GitHub also wants a piece of the cake which is totally fine.

Anyway, keep in mind that GitHub Copilot is [free](https://github.blog/2022-06-21-github-copilot-is-generally-available-to-all-developers/) for students and maintainer of open source projects. Maybe you are one of them.

**Reply by Ghalbeyou** [reply]
Thank you for your answer
